zio-http/src/main/scala/zio/http/service/ServerFastResponseWriter.scala

```diff
@@ -9,22 +10,24 @@ import zio.http.{HExit, Response}
  * written on the channel.
  */
 trait ServerFastResponseWriter[R] { self: ServerInboundHandler[R] =>
-  import ServerInboundHandler.{log, Unsafe}
+  import ServerInboundHandler.log
 
-  def attemptFastWrite(exit: HExit[R, Throwable, Response])(implicit ctx: Ctx): Boolean = {
-    exit match {
-      case HExit.Success(response) =>
-        response.attribute.encoded match {
-          case Some((oResponse, jResponse: FullHttpResponse)) if Unsafe.hasChanged(response, oResponse) =>
-            val djResponse = jResponse.retainedDuplicate()
-            Unsafe.setServerTime(time, response, djResponse)
-            ctx.writeAndFlush(djResponse, ctx.voidPromise()): Unit
-            log.debug("Fast write performed")
-            true
+  def attemptFastWrite(exit: HExit[R, Throwable, Response])(implicit ctx: Ctx): Boolean =
+    Unsafe.unsafe { implicit u =>
```

Comment:
Will cause performance degradation. This interferes with fast path execution.

Reply:
fixed